src/Plugin/Field/FieldFormatter/VisualisationFileFormatter.php

```diff
@@ -16,8 +13,4 @@
  *   }
  * )
  */
-class VisualisationFileFormatter extends FormatterBase {
-
-  use FieldFormatterTrait;
-
-}
+class VisualisationFileFormatter extends VisualisationUrlFormatter { }
```

Comment:
How do you feel about extending both this class and `VisualisationUrlFormatter` from a common class `VisualisationChartFormatter`.

Alternatively, how do you feel about creating a single field formatter that both the URL and File field types use as their default formatters (e.g. `VisualisationChartFormatter`)? I am leaning more towards this option...

Reply:
Good idea! I refactored to be `VisualisationDefaultFormatter` and removed file/url formatters. Feels better having a default that new formatters can extend.